nautobot/utilities/tests/test_templatetags_helpers.py

```diff
@@ -43,14 +44,18 @@ def test_hyperlinked_object(self):
         # An object without get_absolute_url gives a string
         self.assertEqual(hyperlinked_object("hello"), "hello")
         # An object with get_absolute_url gives a hyperlink
-        status = Status.objects.get_for_model(Site).first()
-        site = Site.objects.create(name="Test Site", slug="test-site", status=status)
-        self.assertEqual(hyperlinked_object(site), '<a href="/dcim/sites/test-site/">Test Site</a>')
+        site = Site.objects.first()
+        status = Status.objects.create(name="Testing")
+        status.content_types.set([ContentType.objects.get_for_model(Site)])
+        status.validated_save()
+        site.status = status
+        site.validated_save()
```

Comment:
Why is all this needed? Shouldn't all of our factory-generated sites already have a valid status?

Same question on similar logic below.